tests/CommandsTest.php

```diff
@@ -427,13 +405,35 @@ public function changelogDataProvider()
 
     /**
      * @param string $content
-     * @param array  $expected
+     * @param array  $expectations
      */
-    protected function assertContainsNotContains($content, array $expected)
+    protected function assertContainsNotContains($content, array $expectations)
     {
-        $this->assertContains($expected['contains'], $content);
+        if (!empty($expectations['contains'])) {
+            foreach ((array) $expectations['contains'] as $expected) {
+                $this->assertContains($expected, $content);
+            }
+        }
         if (!empty($expected['not_contains'])) {
-            $this->assertNotContains($expected['not_contains'], $content);
+            foreach ((array) $expectations['not_contains'] as $expected) {
+                $this->assertNotContains($expected, $content);
+            }
+        }
+    }
+
+    /**
+     * @param array $expected
+     */
+    protected function processSettingsAssertions(array $expected)
+    {
+        foreach ($expected as $row) {
+            if (isset($row['file'])) {
+                $content = file_get_contents($this->getSandboxFilepath($row['file']));
+                $this->assertContainsNotContains($content, $row);
```

Comment:
Should it be like:
```suggestion
              $this->assertFileExists($row['file']);
```
?

Reply:
In case the file is given we want to make more inspections than just check if it exists. We can add as many assertions as we want by adding "contains" and "not_contains" entries.

As is will also fail if it's missing because of "file_get_contents", so I skip that test.